app/models/comment.rb

```diff
@@ -71,7 +71,7 @@ def check_for_spam
   validates :comment_content, uniqueness: {
```

Comment:
You can ignore rubocop's `Rails/UniqueValidationWithoutIndex` complaint, we don't need the unique index for now.